### PR TITLE
Modify rule S6865: Updated example, message and removed configuration part for now

### PR DESCRIPTION
**Pull request by navendu-barua-sonarsource**
Removed the example for **service account's configuration** part, as this is not yet supported
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by jonas-wielage-sonarsource — APPROVED** [review]
LGTM

File: rules/S6865/kubernetes/rule.adoc [hunk 1]
```diff
@@ -56,28 +56,6 @@ spec:
 
 ----
 
-The same can be achieved by setting `automountServiceAccountToken: false` in the service account's configuration:
```
Comment (daniel-teuchert-sonarsource) [inline rules/S6865/kubernetes/rule.adoc:59]: Why is this removed?

**Review by daniel-teuchert-sonarsource — CHANGES_REQUESTED** [review]
Overall LGTM. Just a minor question (see below)

File: rules/S6865/kubernetes/rule.adoc (hunk 1 above)
Comment (jonas-wielage-sonarsource) [reply rules/S6865/kubernetes/rule.adoc:59]: Because we are not able to detect if there is a ServiceAccount next to the Pod at the moment.
We will create a follow up ticket to include this, when we have kubernetes cross-file analysis.

File: rules/S6865/kubernetes/rule.adoc [hunk 2]
```diff
@@ -56,28 +56,6 @@ spec:
...
-spec:
-  containers:
-  - name: example-pod
-    image: nginx:1.25.3
-  serviceAccountName: example-service-account
-----
-
 === How does this work?
 
 The automounting of service account tokens can be disabled by setting `automountServiceAccountToken: false` in the pod's specification or in the service account's configuration.
```
Comment (jonas-wielage-sonarsource) [inline rules/S6865/kubernetes/rule.adoc:61]:
```suggestion
The automounting of service account tokens can be disabled by setting `automountServiceAccountToken: false` in the pod's specification. Additionally, it can be disabled in the configuration of an accompanied service account.
```

File: rules/S6865/kubernetes/rule.adoc (hunk 1 above)
Comment (daniel-teuchert-sonarsource) [reply rules/S6865/kubernetes/rule.adoc:59]: I see, thanks for clarifying!

**Review by daniel-teuchert-sonarsource — APPROVED** [review]
LGTM! I suggest applying the suggested change from Jonas.
